sys/src/main/java/com/alibaba/nacos/sys/utils/InetUtils.java

```diff
@@ -156,8 +201,9 @@ public static InetAddress findFirstNonLoopbackAddress() {
                     if (!ignoreInterface(ifc.getDisplayName())) {
                         for (Enumeration<InetAddress> addrs = ifc.getInetAddresses(); addrs.hasMoreElements(); ) {
                             InetAddress address = addrs.nextElement();
-                            boolean isLegalIpVersion = InternetAddressUtil.PREFER_IPV6_ADDRESSES ? address instanceof Inet6Address
-                                    : address instanceof Inet4Address;
+                            boolean isLegalIpVersion =
+                                    InternetAddressUtil.PREFER_IPV6_ADDRESSES ? address instanceof Inet6Address
+                                            : address instanceof Inet4Address;
```

Comment:
Is reformat by nacos-code-style?

Reply:
I checked my idea's configuration, I absolutely use the nacos-code-style.